<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1525</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>852</x>
      <y>200</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1546</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Добавить запись(Создать uniq)</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Изменить на запись(move)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_15"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_16"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>Изменить запись(reset)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_14"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>Поменять записи(swap)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Очистить запись(realese)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_17"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="tableView_2"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Добавить запись(Создать share)</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="text">
              <string>Копировать запись(Создать от share)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_5"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QPushButton" name="pushButton_15">
             <property name="text">
              <string>Создать запись в таблице weak(Создать от share)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_13"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>Изменить запись(reset)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_6"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="pushButton_14">
             <property name="text">
              <string>Занулить запись(reset)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_12"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>Поменять записи(swap)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_3"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_4"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="tableView_3"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="text">
              <string>Копировать запись(Создать от weak)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_7"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="pushButton_13">
             <property name="text">
              <string>Создать запись в таблице share(Создать от weak)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_11"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_11">
             <property name="text">
              <string>Занулить запись(reset)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_8"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="pushButton_12">
             <property name="text">
              <string>Поменять записи(swap)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_9"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_10"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1525</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
